backend/app/services/admin/health_check_service.py

```
@@ -0,0 +1,154 @@
+            latency = (datetime.now() - start).total_seconds() * 1000
+            return ServiceHealth("Weaviate", "unhealthy", round(latency, 2), str(e)[:100])
+
+    async def check_falkordb(self) -> ServiceHealth:
+        start = datetime.now()
+        try:
+            falkor_url = getattr(settings, 'falkordb_url', 'http://localhost:6379')
+            async with aiohttp.ClientSession() as session:
+                async with session.get(
+                    falkor_url,
+                    timeout=aiohttp.ClientTimeout(total=self.timeout)
+                ) as resp:
```

Ruff (0.15.0)

[error] Local variable `resp` is assigned to but never used

Remove assignment to unused variable `resp`

(F841)